src/client/components/Overview/Overview.js

```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import './Overview.styles.css';
 import UserList from '../../containers/UserList/UserList';
 import Search from '../Search/Search';
+import RoomListOverview from '../RoomListOverview/RoomListOverview';
+import PropTypes from 'prop-types';
 import { UserContext } from '../../context/userContext';
 
-function Overview() {
+function Overview(props) {
```

Comment:
As mentioned in App.js, this component should receive no props.
Instead, this function should use `useEffect` to fetch rooms from the backend. If the API is not there, simply pass the `[]` down to the `RoomListOverview` component in line 30.

Reply:
ive removed it and Aswini has the task to connect to backend.